### ISSUE TITLE
Audit .accessibility-aid

### ISSUE THREAD

**Issue by barrymcgee** [ubuntudesign/vanilla-framework#423]
* `.off-left` should be deprecated

* `.accessibility-aid` should be renamed to `.u-invisible`
